Aztec/Classes/TextKit/MediaAttachment.swift

```diff
@@ -28,23 +28,23 @@ open class MediaAttachment: NSTextAttachment {
 
     /// Attributes accessible by the user, for general purposes.
     ///
-    public var extraAttributes = [String: String]()
+    open var extraAttributes = [String: String]()
 
     /// Identifier used to match this attachment with a custom UIView subclass
     ///
     private(set) open var identifier: String
     
     /// Attachment URL
     ///
-    public var url: URL? {
-        didSet {
-            retryCount = 0
-        }
-    }
+    fileprivate(set) open var url: URL?
```

Comment:
Any specific scenario in which we think it may make sense to override this?  I'd consider removing the `open` modifier.